common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_apmanager.yaml

```diff
@@ -184,6 +184,23 @@ cACTION_APMANAGER_STEERING_EVENT_AUTH_FAIL_NOTIFICATION:
   _type: class
   params: sSteeringEvAuthFail
 
+cACTION_APMANAGER_WIFI_CREDENTIALS_SET_REQUEST:
+  _type: class
+  ssid:
+    _type: char    
+    _length: [ "beerocks::message::WIFI_SSID_MAX_LENGTH" ]
```

Comment:
Maybe it was implemented like that before, but this is wrong: the length should be dynamic. So we should have a separate ssid_length field:
```
  ssid_length:
    _type: uint8_t
  ssid:
    _type: char
    _length: [ ssid_length ]
```

Reply:
But add property `_length_var: True` to the `ssid length`:
```
  ssid_length:
    _type: uint8_t
    _length_var: True
  ssid:
    _type: char
    _length: [ ssid_length ]
```

Reply:
I don't think this is required. Check out the var length lists in M2.

Reply:
Changed to list of configdata.